sys-devel/llvm/llvm15-15.0.7.recipe

```diff
@@ -0,0 +1,739 @@
+SUMMARY="Modular and reuseable compiler and toolchain technologies"
+DESCRIPTION="LLVM is a collection of modular and reuseable compiler and and \
+toolchain technologies.The LLVM compiler system for C and C++ includes the \
+following:
+
+- Front-ends for C, C++, Objective-C, Fortran, etc. based on the GCC 4.2 \
+parsers. They support the ANSI-standard C and C++ languages to the same \
+degree that GCC supports them. Additionally, many GCC extensions are \
+supported.
+- A stable implementation of the LLVM instruction set, which serves as both \
+the online and offline code representation, together with assembly (ASCII) \
+and bytecode (binary) readers and writers, and a verifier.
+- A powerful pass-management system that automatically sequences passes \
+(including analysis, transformation, and code-generation passes) based on \
+their dependences, and pipelines them for efficiency.
+- A wide range of global scalar optimizations.
+- A link-time interprocedural optimization framework with a rich set of \
+analyses and transformations, including sophisticated whole-program pointer \
+analysis, call graph construction, and support for profile-guided optimizations.
+- An easily retargettable code generator, which currently supports X86, \
+X86-64, PowerPC, PowerPC-64, ARM, Thumb, SPARC, Alpha, CellSPU, MIPS, MSP430, \
+SystemZ, and XCore.
+- A Just-In-Time (JIT) code generation system, which currently supports X86, \
+X86-64, ARM, AArch64, Mips, SystemZ, PowerPC, and PowerPC-64.
+- Support for generating DWARF debugging information.
+- A C back-end useful for testing and for generating native code on targets \
+other than the ones listed above.
+- A profiling system similar to gprof.
+- A test framework with a number of benchmark codes and applications.
+- APIs and debugging tools to simplify rapid development of LLVM components."
+HOMEPAGE="https://www.llvm.org/"
+COPYRIGHT="2003-2019 University of Illinois at Urbana-Champaign"
```

Comment:
Might need an update to the copyright year

Reply:
llvm doesn't seem to update it, so I'll leave as-is.